### ISSUE TITLE
Correlation Coefficient

### ISSUE THREAD

**Issue by joshenterkine** [dylanmikesell/GEOS397#14]
I'm having some trouble understanding the correlation coefficient matrix output in a few instances.

For example, I have vectors which represent percentages (i.e. sum to one) calculated from the cumsum function:
A = [0.470000000000000;0.530000000000000]
B = [0.222222222222222;0.777777777777778]
C = [0.489795918367347;0.510204081632653]
corrcoef(A, B) =  [1,1;1,1] 
corrcoef(A, C) = [1,1;1,1] 

This has me stymied. They appear to be the same format ( I even cast them as long), but I can't figure out why this is happening, when they are clearly not the same...

**Comment by dylanmikesell** [comment]
Why not try some other similarity metrics...
https://www.mathworks.com/matlabcentral/answers/56096-compare-two-vectors-for-similarity
The distance is another measure...
http://www.analytictech.com/mb876/handouts/distance_and_correlation.htm

**Comment by joshenterkine** [comment]
Good call. I was going to brute-force any values of 1 to be 'NaN' or some other value representing suspicion. 
Thanks!
